### ISSUE TITLE
Internal error @ src/full/Agda/TypeChecking/Primitive.hs:1253

### ISSUE THREAD

**Issue by fredefox** [agda/agda#2943]
Agda gives me:

```
An internal error has occurred. Please report this as a bug.
Location of the error: src/full/Agda/TypeChecking/Primitive.hs:1253
```

But unfortunately no guide on what information to provide.

Steps to reproduce:

```
git clone git@github.com:InitialTypes/Club.git itc
cd itc/lectures/cubical/Exercises/
git checkout aa21d38a8c3d66ee4b6f87fa1c72d5d96d2fe315
agda Abs.agda
```

Produces the above error.

Further information:

```
$ agda --version
> Agda version 2.6.0-5d84754
```

Dependencies:

Cubical:

```
commit 1d6730c4999daa2b04e9dd39faa0791d0b5c3b48 (HEAD -> master, Saizan/master)
Author: Frederik Hanghøj Iversen <fhi.1990@gmail.com>
Date:   Tue Jan 30 20:55:35 2018 +0100
```

Standard Library:

```
commit b5bfbc3c170b0bd0c9aaac1b4d4b3f9b06832bf6 (HEAD -> master, origin/master, origin/HEAD)
Merge: 26c3eca5 751f8f56
Author: Nils Anders Danielsson <nad@cse.gu.se>
Date:   Tue Jan 30 20:24:46 2018 +0100
```

**Comment by fredefox** [comment]
I'm installing the most recent version of Agda to see if it removes the error (old one was all of 2 weeks old). I think not though, since the line where the error originates from is also on `master` just a few lines down from where I reported.

**Comment by fredefox** [comment]
Fixed in `2.6.0-5d84754-dirty` (it wasn't dirty tho)
